app/jobs/subscription_placement_job.rb

```diff
@@ -30,6 +30,8 @@ def proxy_orders
   def place_order_for(proxy_order)
     JobLogger.logger.info("Placing Order for Proxy Order #{proxy_order.id}")
     initialise_order(proxy_order)
```

Comment:
if there was an issue while creating the order within `ProxyOrder#initialise_order!`, given it calls `save!`, we should have received an error in Bugsnag because `StandardError` exceptions are rescued. Didn't you find any for this issue?